src/app/DTT/business_locations/business_ask.js

```diff
@@ -0,0 +1,89 @@
+import { Box, Stack } from '@chakra-ui/react'
+import { isEmpty, lensProp, set } from 'ramda'
+import { useEffect, useState } from 'react'
+
+import RepeatableAsk from 'app/ASKS/repeatable-ask'
+import safelyParseJson from 'utils/helpers/safely-parse-json'
+import { useIsMobile } from 'utils/hooks'
+import AddressPicker from '../address/address_picker'
+import Text from '../text'
+
+const BusinessAsk = ({
+  questionCode,
+  onSendAnswer,
+  data,
+  regexPattern,
+  errorMessage,
+  parentCode,
+  placeholderName,
+  attributeCode,
+  targetCode,
+  clientId,
+}) => {
+  const isMobile = useIsMobile()
+  const [value, setValue] = useState(safelyParseJson(data?.value), {})
```

Comment:
why are we initializing useState with two values of data and an empty object?

Reply:
ah, that was meant to be safelyParseJson(data?.value, {})). Either way it's uneeded as safelyParseJson returns an empty object on fail anyway